### DISCUSSION TITLE
Terms.php not found

### DISCUSSION THREAD

**Discussion by musokeharun** [statamic/cms#9231] [Troubleshooting]
Hey Community,

Thank you the crucial effort you put forward.

However i have encountered this error quite lately. 
include(/statamic/cms/src/Fieldtypes/Terms.php): Failed to open stream: No such file or directory

Thank you very much

**Comment by duncanmcclean** [comment]
Please provide the full stack trace.

**Comment by musokeharun** [comment]
https://flareapp.io/share/V7j9zA3P

Please find it there,

**Reply by duncanmcclean** [reply]
Does the file actually exist? It should be located in `vendor/statamic/cms/src/Fieldtypes`

**Reply by musokeharun** [reply]
It auto deletes every time i install with composer on the server.
It doesnt exist, even if i upload it

**Reply by musokeharun** [reply]
Hey @duncanmcclean,

This challenge is persisting.
Would you care to extend a bit of advice

Thank you

**Comment by musokeharun** [comment]
Hey @duncanmcclean,

This challenge is persisting.
Would you care to extend a bit of advice

Thank you.

**Reply by duncanmcclean** [reply]
What does your deployment process look like?

Composer shouldn't be doing anything to delete files after they've been installed.
